### PR TITLE
Dev#youngestdev

### PR DESCRIPTION
**Pull request by Youngestdev**
Added `.gitignore` due to editor and other stuffs from my end.

This PR contains the following modifications:

1. Package renaming - The linter I used informed me of package naming. Are you okay with this change?
2. Added the methods for country, city and state retrieval respectively. However, the methods that should return multiple values sharing the same IDs such as `GetCities` return single elements which I suspect is a case of proper looping - I'm not sure of how to really implement this.
3. Added `helpers.go` - this file here contains functions ( private methods ) that load the json files required for specific tasks.

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package golocation
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func loadCountries() []byte {
+	file, err := os.Open("/home/Youngestev/go/bin/src/github.com/Youngestdev/go-location/db/countries.json")
```
Comment (ichtrojan) [inline helpers.go]: Change this to the local directory ./db/countries.json

File: helpers.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	countries, _ := ioutil.ReadAll(file)
+	return countries
+}
+
+func loadStates() []byte {
+	file, err := os.Open("/home/Youngestev/go/bin/src/github.com/Youngestdev/go-location/db/states.json")
```
Comment (ichtrojan) [inline helpers.go]: Change this to the local directory ./db/states.json

File: helpers.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	states, _ := ioutil.ReadAll(file)
+	return states
+}
+
+func loadCities() []byte {
+	file, err := os.Open("/home/Youngestev/go/bin/src/github.com/Youngestdev/go-location/db/cities.json")
```
Comment (ichtrojan) [inline helpers.go]: Change this to the local directory ./db/states.json

**Comment by Youngestdev** [comment]
@ichtrojan... There's no comment as you tweeted 🙂

**Review by ichtrojan — APPROVED** [review]

**Pull request merged**